.github/workflows/test.yml

```diff
@@ -104,14 +104,5 @@ jobs:
         run: |
           gem install bundler -v '~> 1.17.3'
           bundle update
```

Comment:
Like the other PR: explicit bundler version + bundle update isn't needed anymore.